### PR TITLE
Upgrade to SDK v7 + Package rename

### PR DESCRIPTION
**Pull request by Simply007**
### Motivation

Upgrade to Kentico Cloud SDK v7 (@kentico/kontent-delivery) + rename Kentico Cloud to Kentico Kontent

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

Click through the site if every breaking change was handled.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +17,7 @@
   },
   "dependencies": {
     "dateformat": "^3.0.3",
-    "kentico-cloud-delivery": "^5.7.2",
+    "@kentico/kontent-delivery": "^7.0.1",
```
Comment (OnyxPrime) [inline package.json]: We should update the PR title to `SDK v7`, instead of `SDK v6`

File: src/components/Admin/Configuration.vue [hunk 2]
```diff
@@ -203,41 +203,48 @@ export default {
     redirectToHome(newProjectId) {
       const infoMessage =
         newProjectId === defaultProjectId
-          ? 'You\'ve configured your app to with a project ID of a shared Kentico Cloud project.'
-          : `You've configured your app with a project ID "${newProjectId}". You can edit its contents at https://app.kenticocloud.com/.`;
+          ? 'You\'ve configured your app to with a project ID of a shared Kentico Kontent project.'
+          : `You've configured your app with a project ID "${newProjectId}". You can edit its contents at https://app.konent.ai/.`;
```
Comment (Enngage) [inline src/components/Admin/Configuration.vue]: Typo here :-)

File: src/components/Admin/Configuration.vue [hunk 3]
```diff
@@ -203,41 +203,48 @@ export default {
...
       this.$router.push(`/?infoMessage=${infoMessage}`);
     },
     receiveMessage(event) {
-      if (event.origin.toLowerCase() !== 'https://app.kenticocloud.com')
-        return;
+      if (event.origin.toLowerCase() !== 'https://app.konent.ai') return;
```
Comment (Enngage) [inline src/components/Admin/Configuration.vue]: And here

File: README.md [hunk 4]
```diff
@@ -51,25 +52,25 @@ When you now run the application, you will see all project content including the
 
 ## Content administration
 
-1. Navigate to <https://app.kenticocloud.com> in your browser.
+1. Navigate to <https://app.konent.ai> in your browser.
```
Comment (Enngage) [inline README.md]: Typo

Comment (Simply007) [inline README.md]:
```suggestion
1. Navigate to <https://app.kontent.ai> in your browser.
```

File: src/components/Admin/Configuration.vue (hunk 2 above)
Comment (Simply007) [inline src/components/Admin/Configuration.vue]:
```suggestion
          : `You've configured your app with a project ID "${newProjectId}". You can edit its contents at https://app.kontent.ai/.`;
```

File: src/components/Admin/Configuration.vue (hunk 3 above)
Comment (Simply007) [inline src/components/Admin/Configuration.vue]:
```suggestion
      if (event.origin.toLowerCase() !== 'https://app.kontent.ai') return;
```

**Pull request merged**